optional/vtd/vendor/vtd-2022.3/entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+
+# This entrypoint script wraps VTD like it would be running in the foreground
+# VTD's startscript actually terminates itself and the main simulator proceeses
+# are regularly a bunch of orphans.
+# In order to compensate for this misbehavior, this script monitor's the state
+# of the most important process, the simServer. Its termination leads to the
+# container's termination.
+# All arguments are passed unchanged to the vtdStart.sh script.
```

Comment:
```suggestion
# This entrypoint script wraps VTD so it is as if it were running in the foreground.
#
# VTD's start script terminates itself, so that the main simulator processes
# are regularly a bunch of orphans (zombies).
# In order to compensate for this misbehavior, this script monitors the state
# of the most important process, the simServer. Its termination leads to the
# container's termination.
#
# All arguments are passed unchanged to the vtdStart.sh script.
```